lib/MusicBrainz/Server/Form/Utils.pm

```diff
@@ -208,7 +208,7 @@ sub validate_username {
     my $previous_username = $self->init_value;
 
     if (defined $username) {
-        unless (defined $previous_username && $previous_username eq $username) {
+        unless (defined $previous_username && uc($previous_username) eq uc($username)) {
```

Comment:
The only possible problem I see here is that `uc($previous_username) eq uc($username)` in Perl may not be equivalent to `lower(x) = lower(y)` in Postgres. `uc` is locale-aware but our database cluster uses `--no-locale`, so they may not behave the same for Unicode characters...perhaps we should add something like `are_names_equivalent` to `Data::Editor` that just passes both names to a query like `SELECT lower(?) = lower(?)`?

Reply:
Ok, made that change to be fully safe :)